README.md

```diff
@@ -5,3 +5,13 @@ This role deploys [Storage node][neofs-node] for [NeoFS][neofs].
 
 [neofs-node]: https://github.com/nspcc-dev/neofs-node
 [neofs]:      https://fs.neo.org
+
+### NeoFS Storage Compatibility Table
+
+| NeoFS Node version | ansible-neofs-storage role version | min NeoFS Node supported version | max NeoFS Node compatible version |
```

Comment:
Why do we have _three_ NeoFS node versions?

Reply:
A given role tag introduces the updates for the "NeoFS Node version", "min" and "max" versions correspond to minimal supported and maximum compatible versions for this tag.

Reply:
OK, let's have it this way for now. Can be fixed later if any better ideas appear.